accepted/0000-timezones.md

```diff
@@ -0,0 +1,102 @@
+- Feature Name: timezones
+- Start Date: 2019-09-16
+- RFC PR: 
+
+# Summary
+[summary]: #summary
+
+Timezones can be a big problem when managing an application like Uyuni that can be deployed and accessed from anywhere in the world. 
+If we don’t have some rules on how to display and handle dates, this can become quite a big mess to manage. 
+
+
+# Motivation
+[motivation]: #motivation
+
+The goal of this RFC is to define those rules and serve as a reference for the future.
+
+These rules should define a consistent way of display dates on the UI and make this information as clear as possible for the user.
+
+The user should be able to easily understand in which timezone a date is displayed and aware if there is any difference from the timezone
+he is accessing from.
+
+# Glossary
+[glossary]: #glossary
+
+When talking about uyuni there are 4 different Timezones contexts that are important to understand and distinguish:
+
+* *Managed systems timezone:* the timezone of each minion/client. This doesn't have any impact on this RFC, but it's important to mention.
+
+* *Uyuni Server timezone:* the configured timezone in the server where uyuni is deployed 
+
+* *User preferences timezone:* the defined timezone on the user preferences settings. Here we should be able to choose the timezone, date format (yyyyddmm) and the time format (12/24 hours).
+
+* *Client browser timezone:* the browser/pc timezone where the client is accessing from. This timezone shouldn’t affect any displayed date, but the user should be aware of it.
+
+
+# Detailed design
+[design]: #detailed-design
+
+There are 3 topics that we need to have in consideration here:
+
+## Display of dates
+
+The display of dates should always take in consideration the 3 user preferences settings: timezone, date-format and time-format.
+
+**For example:** '2019-09-10 13:00:00 **CEST**'
+
+At the moment only the setting timezone is implemented. The property date-format and time-format could be added on a second iteration, 
```

Comment:
nice to know!

hum since we don't know yet when we will add the localization (language) again, it seems more feasible to add these new configuration props instead of inferring them from the location country. What do you think?

Nevertheless, the new implementation could take the old comment code as an inspiration :)

Reply:
> nice to know!
> 
> hum since we don't know yet when we will add the localization (language) again, it seems more feasible to add these new configuration props instead of inferring them from the location country. What do you think?
> 
> Nevertheless, the new implementation could take the old commented code as an inspiration :)

We will add localization for 4.1, so please, take this into account.

Reply:
Internationalization (as in being ready for adding localization files that come from our translators has been a must-have feature in 4.0! The team doesn't have to care about translations, but the complete UI needs to be _enabled_ for localization. I'm not sure where the disconnect was, but this was always clearly communicated by product management and also part of the engineering interlock.